test/e2e/framework/util.go

```diff
@@ -128,8 +128,29 @@ const (
 
 	// How long claims have to become dynamically provisioned
 	ClaimProvisionTimeout = 5 * time.Minute
+
+	// When these values are updated, also update cmd/kubelet/app/options/options.go
+	defaultPodInfraContainerImageName    = "gcr.io/google_containers/pause"
+	defaultPodInfraContainerImageVersion = "3.0"
 )
 
+// GetPauseImageName fetches the architecture of the target cluster and chooses the pause image to use
+func GetPauseImageName(c *client.Client) string {
+	arch := ""
+	sVer, err := c.Discovery().ServerVersion()
+	if err != nil {
+		// If we failed to get the server version for some reason, default to amd64
+		arch = "amd64"
+	} else {
+		// Split the platform string into OS and Arch separately.
+		// The platform string may for example be "linux/amd64", "linux/arm" or "windows/amd64"
+		osArchArray := strings.Split(sVer.Platform, "/")
```

Comment:
probably need to handle `sVer.Platform` being empty - this can happen in skew testing with new e2es against an older cluster, for example.